_dashboards/get-started/quickstart-dashboards.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Quickstart guide for OpenSearch Dashboards
+nav_order: 20
+has_children: false
+---
+
+# Quickstart guide for OpenSearch Dashboards
+
+This quickstart guide covers the core concepts that you need to understand to get started with OpenSearch Dashboards. You'll learn how to:
+
+- Add sample data.
+- Explore and inspect data with **Discover**.
+- Visualize data with **Dashboard**.
+- Add sample data.
+- Explore and inspect data with **Discover**.
+- Visualize data with **Dashboard**.
+
+Before you get started, make sure you've installed OpenSearch and OpenSearch Dashboards. For information on installation and configuration, see [Install and configure OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/) and [Install and configure OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/index/).
+{: .note}
+
+# Adding sample data
+
+Sample datasets come with visualizations, dashboards, and other tools to help you explore Dashboards before you add your own data. To add sample data, perform the following steps:
+
+1. Verify access to OpenSearch Dashboards by connecting to [http://localhost:5601](http://localhost:5601) from a browser. The default username and password are `admin`.
+1. On the OpenSearch Dashboards **Home** page, choose **Add sample data**.
+1. Choose **Add data** to add the data sets, as shown in the following image.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/add-sample-data.png" alt="Sample data sets" width="800">
+
+# Exploring and inspecting data
+
+In [**Discover**](discover/index.md), you can: 
+
+- Choose data to explore, set a time range for that data, search it using [Dashboards Query Language (DQL)]({{site.url}}{{site.baseurl}}/dashboards/dql/), and filter the results.
+- Explore the data, view individual documents, and create tables summarizing the data's contents.
+- Visualize your findings.
+
+## Try it: Getting familiar with Discover
+
+1. On the OpenSearch Dashboards **Home** page, choose **Discover.**
```

Comment:
Does "Discover" include a period in the UI? If not, the period should not be in bold.

Reply:
Change made